bam_masterdata/cli/cli.py

```diff
@@ -727,7 +728,14 @@ def run_parser(
     required=True,
     help="OpenBIS collection name",
 )
-def parser(files_parser, project_name, collection_name):
+@click.option(
+    "--space_name",
```

Comment:
I would keep hyphens instead of underscores: `--space-name` (note you have an underscore between "space" and "name")